examples/synthesis.md

```diff
@@ -0,0 +1,228 @@
+    pass
+```
+
+The `None` arguments signals here, that we read the token from the environment as discussed above.
+You may notice, that we are not executing any code inside the context manager block. If you were to write a python script, compared to a jupyter notebook, you would add all of the following code inside that block.
+Here in a jupyter notebook, we need to connect manually. We just have to remember to disconnect at the end.
+
+```python
+api = cript.API("http://development.api.mycriptapp.org/", None)
+api = api.connect()
+```
+
+# Create a Project
+
+All data uploaded to CRIPT must be associated with a `Project` node. `Project` can be thought of as an overarching research goal. For example finding a replacement for an existing material from a sustainable feedstock.
+
+```python
+# create a new project in the CRIPT database
+project = cript.Project(name="My first project.")
+```
+
+# Create a Collection node
+
+For this project you can create multiple collections, which represent a set of experiments.
+For example, you can create a collection for a specific manuscript, or you can create a collection for initial screening of candidates and one for later refinements etc.
+
+So, let's create a collection node and add it to the project.
+
+```python
+collection = cript.Collection(name="Initial screening")
+# We add this collection to the project as a list.
+project.collection += [collection]
+```
+
+Note, that if you are interested into the inner workings of CRIPT, you can obtain a JSON representation of your data graph at any time.
+
+```python
+print(project.json)
+print("\nOr more pretty\n")
+print(project.get_json(indent=2).json)
+```
+
+# Create an Experiment node
+
+The collection holds a series of `Experiment` nodes.
+Every experiment holds processes and data related this particular experiment.
+
+And we can add this experiment to the collection of the project.
+
+```python
+experiment = cript.Experiment(name="Anionic Polymerization of Styrene with SecBuLi")
+collection.experiment += [experiment]
+```
+
+# Create an Inventory
+
+An `Inventory` contains materials, that are well known and usually not of polymeric nature.
+The are for example the chemical you buy commercially and use as input into your synthesis.
+
+For this we create this inventory by adding the `Material`s we need one by one.
+
+```python
+solution = cript.Material(name="SecBuLi solution 1.4M cHex", identifiers=[{"smiles": "[Li]C(C)CC"}])
+```
+
+These materials are simple, notice how we use the SMILES notation here as an identifier for the material.
+Similarly, we can create more initial materials.
+
+```python
+toluene = cript.Material(name="toluene", identifiers=[{"smiles": "Cc1ccccc1"}])
+styrene = cript.Material(name="styrene", identifiers=[{"smiles": "c1ccccc1C=C"}])
+butanol = cript.Material(name="1-butanol", identifiers=[{"smiles": "OCCCC"}])
+methanol = cript.Material(name="methanol", identifiers=[{"smiles": "CO"}])
```

Comment:
if possible see if you can have a different identifier for each material so we can show off all of our different identifiers

```
bigsmiles,
smiles,
pubchem_cid,
chemical_id,
chem_repeat,
````

I would pick whichever that is most popular, so users know that we are compatible with their favorite way of identifying materials and maybe something that CRIPT specifically does, so they know there are more possibilities and the things they've wanted to do already exists and they can start using CRIPT for it

Reply:
Yes, agreed.
However, the identifiers don't quite make sense for these materials.
All of them should have `smiles`, we can add more to show them off.

Except for bigSMILES, since these are not polymers, bigSMILES and smiles are identical.

Reply:
I added those.
And actually that was very good, because I used the wrong identifier for the solution.
BTW, if you ever have to work with this: https://cactus.nci.nih.gov/chemical/structure
super helpful to convert between different identifiers.

(Maybe we should even include something, that suggests identifiers, automatically from this service.)

Reply:
That website is super helpful! I will save it somewhere. I wonder if it would be a good idea to put it within our `#cript-dev` channel and pin it so everyone has access to it for future development, or just put it within our wiki to use for later